README.md

```diff
@@ -305,6 +306,30 @@ method takes one argument:
 
 2. `chain` must return a value of the same Chain
 
+### ChainRec
+
+A value that implements the ChainRec specification must also implement the Chain specification.
+
+1. `T.chainRec((done, next, value) => g(done(value)), i) `is equivalent to `g(i)`
```

Comment:
There is something wrong with `g`. It's certainly takes different types in `g(done(value))` and `g(i)` invocations.

Maybe you meant  `T.chainRec((done, next, value) => T.of(done(g(value))), i)`?